### PR TITLE
document connecting to an NLU only server

### PR DESCRIPTION
**Pull request by melindaloubser1**
**Proposed changes**:
- Add documentation for connecting a dialogue-management only server with a separately running NLU server
- closes https://github.com/RasaHQ/rasa/issues/7376

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by akelad** [comment]
@melindaloubser1 did you check that it still behaves as expected?

**Comment by akelad** [comment]
i'm also not sure this belongs in the NLU only section 🤔 you'd usually use that if you're running the dialogue manager and NLU separately

**Comment by melindaloubser1** [comment]
Yes, it does behave as expected. I wasn't really sure where to put it since there isn't an "endpoints" page specifically - should it go under the http api page?

**Comment by akelad** [comment]
@erohmensing i feel like you probably know better where it should go?

File: docs/docs/nlu-only.mdx [hunk 1]
```diff
@@ -51,3 +51,31 @@ To do this, run:
...
+by adding the connection details to the dialogue management server's endpoint configuration file:
+
+```yaml-rasa title="endpoints.yml"
+nlu:
+    url: "http://<your nlu host>:<your nlu port>"
+    token: <token>  # [optional]
+    token_name: <name of the token> # [optional] (default: token)
+```
+
+The `token` and `token_name` refer to optional [authentication paramenters](./http-api.mdx#token-based-auth).
```
Comment (melindaloubser1) [inline docs/docs/nlu-only.mdx:68]:
```suggestion
The `token` and `token_name` refer to optional [authentication parameters](./http-api.mdx#token-based-auth).
```

File: docs/docs/nlu-only.mdx [hunk 2]
```diff
@@ -51,3 +51,31 @@ To do this, run:
...
+    url: "http://<your nlu host>:<your nlu port>"
+    token: <token>  # [optional]
+    token_name: <name of the token> # [optional] (default: token)
+```
+
+The `token` and `token_name` refer to optional [authentication parameters](./http-api.mdx#token-based-auth).
+
+The dialogue management server should serve a model that does not include an NLU model.
+To obtain a dialogue management only model, train a model with `rasa train core` or use 
+`rasa train` but exclude all NLU data.
```
Comment (erohmensing) [inline docs/docs/nlu-only.mdx:72]: I would put this part at the top where we introduce the "dialogue management only server", or create another short header explaining what that is and how to spin one up

File: docs/docs/nlu-only.mdx [hunk 3]
```diff
@@ -51,3 +51,31 @@ To do this, run:
...
+
+When the dialogue management server receives a message, it will [send a request](https://rasa.com/docs/rasa/pages/http-api#operation/parseModelMessage) to 
+`http://<your nlu host>:<your nlu port>/model/parse` and use the parsing information returned.
+
+Note that you should **not** use the same endpoint configuration file for the NLU server as for the dialogue management server, since
+the `nlu` endpoint refers back to the NLU server itself.
```
Comment (erohmensing) [inline docs/docs/nlu-only.mdx:78]: This is a bit confusing to me as written -- but if you provide it does it get mad? I would think NLU-only doesn't even load up endpoints 🤔

Comment (melindaloubser1) [reply docs/docs/nlu-only.mdx:78]: It does get mad, because it tries to circular-reference itself - which is very confusing. I'll add a better explanation

**Comment by melindaloubser1** [comment]
Moved it to its own page under API

File: docs/docs/nlu-only-server.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+id: nlu-only-server
+sidebar_label: NLU Only Server
+title: NLU Only Server
+description: Read about connecting to a Rasa NLU only server using the HTTP API.
+abstract: You can run an NLU only server and use the HTTP API to connect to it.
```
Comment (erohmensing) [inline docs/docs/nlu-only-server.mdx:6]:
```suggestion
sidebar_label: NLU-Only Server
title: NLU-Only Server
description: Read about connecting to a Rasa NLU-only server using the HTTP API.
abstract: You can run an NLU-only server and use the HTTP API to connect to it.
```

File: docs/docs/nlu-only-server.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+The dialogue management server should serve a model that does not include an NLU model.
+To obtain a dialogue management only model, train a model with `rasa train core` or use 
+`rasa train` but exclude all NLU data.
+
+When the dialogue management server receives a message, it will [send a request](https://rasa.com/docs/rasa/pages/http-api#operation/parseModelMessage) to 
+`http://<your nlu host>:<your nlu port>/model/parse` and use the parsing information returned.
+
+:::note endpoint configuration
+The endpoint configuration for the dialogue management server will include an `nlu` endpoint that refers to your NLU only server. Therefore you should **use a separate endpoint configuration file** for the NLU server, excluding the `nlu` endpoint.
+:::
```
Comment (erohmensing) [inline docs/docs/nlu-only-server.mdx:33]: Much more clear, thanks!

**Review by erohmensing — APPROVED** [review]

**Comment by akelad** [comment]
@melindaloubser1 can this be merged?

**Comment by melindaloubser1** [comment]
In general yes, but looks like there's a validation error that prevented auto-merging; I'll look into it

**Pull request merged**